entry_types/scrolled/package/src/frontend/playerControls/ControlBar.js

```diff
@@ -0,0 +1,78 @@
+import React, {useState} from 'react';
+import classNames from 'classnames';
+
+import PlayIcon from '../assets/images/playerControls/play_arrow_24px.svg';
+import PauseIcon from '../assets/images/playerControls/pause_24px.svg';
+import SubtitlesIcon from '../assets/images/playerControls/subtitles_24px.svg';
+import SettingsIcon from '../assets/images/playerControls/settings_24px.svg';
+
+import {ProgressIndicators} from './ProgressIndicators'
+import {TimeDisplay} from './TimeDisplay'
+import {ContextMenu} from './ContextMenu'
+
+import styles from './ControlBar.module.css';
+import themeTransparent from './themes/ControlBarThemeTransparent.module.css';
+import themeWhite from './themes/ControlBarThemeWhite.module.css';
+import themeBlack from './themes/ControlBarThemeBlack.module.css';
+
+export function ControlBar(props) {
+  const [settingsMenuHidden, setSettingsMenuHidden] = useState(props.settingsMenuHidden);
+  const [subtitlesMenuHidden, setSubtitlesMenuHidden] = useState(props.subtitlesMenuHidden);
+
+  const theme = {
+    transparent: themeTransparent,
+    white: themeWhite,
+    black: themeBlack
+  }[props.appearance];
```

Comment:
Since themes already are  a concept in Pageflow, I'd name this something else.

Reply:
I went with `style` or now. This might be confusing with our css-module integration, but `theme` and `appearance` are also keywords which are already taken in the codebase

Reply:
after integrating the controls it turned out that the transparent version with white or black buttons looks best, IMO, so I removed the theme/style/appearance altogether for now and just went with `white` or `black`